spot-client/src/common/app-state/spot-tv/actions.js

```diff
@@ -41,3 +41,19 @@ export function setSpotTVState(newState) {
         newState
     };
 }
+
+/* eslint-disable jsdoc/require-description-complete-sentence */
+/**
+ * Extracts a phone number(if any) from the invites passed to the meeting view as part of the URL query and stores it in
+ * Spot TV state so that Spot Remotes can display it in the UI.
+ *
+ * @param {Array<{ type: string, number: string }>} [invites] - An array of invites as specified by Jitsi Meet iframe
+ * API.
+ * @returns {Object}
+ */
+export function storePhoneNumberFromInvites(invites) {
+    const phone = invites?.find(invite => invite.type === 'phone')?.number;
+
+    return setSpotTVState({ invitedPhoneNumber: phone });
```

Comment:
You want this to be set to undefined if no invite is found, correct?

Reply:
yes